CRM/Core/Smarty.php

```diff
@@ -300,11 +300,11 @@ public function appendValue($name, $value) {
   }
 
   public function clearTemplateVars() {
-    foreach (array_keys($this->_tpl_vars) as $key) {
+    foreach (array_keys($this->get_template_vars()) as $key) {
```

Comment:
I don't think it is a problem in this class FWIWI

Reply:
Does it hurt to change it though? Haven't really looked at it in detail.

Reply:
nah I don't think it hurts